tests/test_load_datasets.py

```diff
@@ -47,3 +47,7 @@ def test_random_name(self):
     def test_wrong_hash(self):
         with self.assertRaises(ConnectionError):
             ACDC('wrong_hash_test').load()
+
+    def test_check_miccai15(self):
```

Comment:
I do not understand what certainly this test do. There are no other tests with signature `test_check_`. If this download the dataset, I suppose this is not a good idea, because we already check functions on the ACDC dateset. Loading of the additional data causes performance slowdown for tests.

Reply:
fixed